### DISCUSSION TITLE
Editieren von Inhalten - Validation wie in WebGIS4

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#57] [Q&A]
Gibt es es die Option aus WebGIS4 noch, eine Eingabe auf ihre räumliche Korrektheit abhängig von einen Rollenparameter zu prüfen?
<edit:validations>
                  <edit:validation 
                            field="gem_nr" 
                            operator="in" 
                            validator="role-parameter:GKZ" 
                            message="Die Fläche überlappt ein fremdes Gemeindegebiet! Bitte überprüfen Sie die Fläche." 
                            />
</edit:validations>

**Comment by faulgurke** [comment]
Ergänzung: Insbesondere dann, wenn mehrere Nutzer im gleichen Datenbestand Editieren und die Sichtbarkeit über einen Darstellungsfilter gesteuert wird, ist diese Option Gold wert.

**Comment by gis-eni** [comment]
Die Funktion wurde bisher nicht ins WebGIS 5 überführt.
Kommt wieder mit Build 5.23.2601. Dann gibt es ebenfalls die Möglichkeit, diese Validierung im CMS zu parametrieren.

**Reply by faulgurke** [reply]
Ich finde die Validierungseinstellung im CMS nicht (Version: 5.23.2802).
Tipp?

**Comment by gis-eni** [comment]
Eventuell musst du noch ein "Reload Schema" (links in der Werkzeugliste) ausführen.

**Reply by faulgurke** [reply]
Das war's - Danke!
